### DISCUSSION TITLE
Get first post (including date and time)

### DISCUSSION THREAD

**Discussion by mikenguyen13** [davidteather/TikTok-Api#857] [Ideas]
Hi David, 
I am interested in getting the first post's date of an account (I assume Selenium could help) to estimate the age of that account. I was wondering if you think it's possible to add this feature into the package. 

Thank you.

**Comment by HeLinSpace** [comment]
when i add user  in https://tikapi.io/developer   i got this message 'Too many attempts. Try again later.' all the time

**Reply by HeLinSpace** [reply]
![image](https://user-images.githubusercontent.com/38428011/159219201-2874ce05-6064-4a78-b784-c55d71637068.png)

**Reply by davidteather** [reply]
TikAPI is **NOT** affiliated with TikTokAPI.
TikTokAPI will remain free, TikAPI is a sponsor of the package and pays for advertising

**Comment by davidteather** [comment]
The best you could do is iterate over a user's videos until the video runs out and find the smallest `create_time`, but there's a limit to the # of videos you can query so for large accounts you probably couldn't get the last video.
